terasoluna-tourreservation-selenium/src/test/java/org/terasoluna/tourreservation/tourreserve/selenium/WaitWebDriverEventListener.java

```diff
@@ -94,7 +94,8 @@ public void beforeScript(String script, WebDriver driver) {
 
     @Override
     public void afterScript(String script, WebDriver driver) {
-        waitPageLoading(driver);
+        if (script.contains("click"))
```

Comment:
- [ ] It seems better to determine using `contains(".click(")`.

`jse.executeScript("document.getElementById('clickedvalue')").getText()` -> avoid wait.
`jse.executeScript("document.getElementById('btn').click()")` -> enable wait.
`jse.executeScript("document.getElementById('btn').click(function() {...})")` -> enable wait.